### DISCUSSION TITLE
与tailwindcss一同使用时，样式优先级问题

### DISCUSSION THREAD

**Discussion by kkfive** [3lang3/react-vant#736] [Q&A]
场景：
我想通过直接传入 `h-[20px]` 来修改按钮的高度，但是这个属性并未生效
![image](https://github.com/3lang3/react-vant/assets/82026204/d7064b7d-1946-4e4d-b864-44896455066c)

这个是我的代码 
![image](https://github.com/3lang3/react-vant/assets/82026204/8b7755c9-8dd2-43f5-8a68-c6e531f3eb12)

**Comment by dmwin72015** [comment]
我也知道，怎么办， 我是用@layer import 也不行 怎么办呢！
